cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/model/SortedReplicasTest.java

```diff
@@ -70,7 +71,7 @@ public void testLazyInitialization() {
     assertEquals("The replicas should be sorted lazily", 0, sr.numReplicas());
     sr.remove(replica);
     assertEquals("The replicas should be sorted lazily", 0, sr.numReplicas());
-    NavigableSet<ReplicaWrapper> sortedReplicas = sr.sortedReplicas();
+    List<Replica> sortedReplicas = sr.sortedReplicas();
```

Comment:
Nit: I believe here we were testing `sortedReplicaWrappers` after the rename -- i.e. not `sortedReplicas` -- hence, we do not need to import `List` for this change.